### PR TITLE
Rename StatusIcon to StatusIndicator

### PR DESCRIPTION
**Pull request by el-dav**

### CODE REVIEW

**Review by joshwooding — CHANGES_REQUESTED** [review]
Needs a changeset

Might not be possible but it would be nice to see if we can get git to track the rename of the files

File: packages/core/stories/status-indicator.doc.stories.mdx [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+
+## Characteristics Used
+
+<CharacteristicUsage of={StatusIndicator} />
+
+## Related components
+
+The status indicator component is based on the following component:
+
+- [Indicator](?path=/story/documentation-styles-and-theming-foundations-indicatorography--page)
```
Comment (JoanaMMoreira) [inline packages/core/stories/status-indicator.doc.stories.mdx:77]: Should this be `Icon`?

Comment (joshwooding) [reply packages/core/stories/status-indicator.doc.stories.mdx:77]: The link should be corrected too

**Review by joshwooding — DISMISSED** [review]

File: packages/core/src/form-field/StatusIndicator.tsx [hunk 2]
```diff
@@ -1,13 +1,14 @@
 import { IconProps } from "@jpmorganchase/uitk-icons";
 import cx from "classnames";
 import { ElementType } from "react";
-import { StatusIcon } from "../status-icon";
+import { StatusIndicator as BaseStatusIndicator } from "../status-indicator";
```
Comment (joshwooding) [inline packages/core/src/form-field/StatusIndicator.tsx:4]: @Fercas123

**Review by JoanaMMoreira — APPROVED** [review]

File: packages/core/stories/status-indicator.doc.stories.mdx (hunk 1 above)
Comment (el-dav) [reply packages/core/stories/status-indicator.doc.stories.mdx]: Well spotted

File: packages/core/stories/status-indicator.doc.stories.mdx [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+
+## Characteristics Used
+
+<CharacteristicUsage of={StatusIndicator} />
+
+## Related components
+
+The status indicator component is based on the following component:
+
+- [icon](?path=/story/documentation-styles-and-theming-foundations-iconography--page)
```
Comment (sion-docs) [inline packages/core/stories/status-indicator.doc.stories.mdx:77]: Is there a reason icon isn't capitalized?

Comment (joshwooding) [reply packages/core/stories/status-indicator.doc.stories.mdx:77]: It should be

**Pull request merged**